### DISCUSSION TITLE
Fallbacks

### DISCUSSION THREAD

**Discussion by MaxDonchenko** [vanilla-extract-css/vanilla-extract#532] [Q&A]
Hey 👋
Do you plan to support [fallback styles](https://cssinjs.org/jss-syntax/?v=v10.5.0#fallbacks)?

If I get it right, the `fallbackVar` [utility](https://github.com/seek-oss/vanilla-extract#fallbackvar) from vanilla-extract is a bit different thing. Correct me if I'm wrong:

![image](https://user-images.githubusercontent.com/10671879/146457598-7c94f438-785a-4153-a342-0a9ee2483176.png)
![image](https://user-images.githubusercontent.com/10671879/146457635-eacc4836-c298-435a-83da-7d2b6d379bf9.png)

**Comment by mattcompiles** [comment]
Hey mate 👋 

Sorry, just realised this isn't documented but you can already do this using arrays. And you're correct, `fallbackVar` is designed just for CSS variable usage. 

Example:
```js
export const scrollbarWrapper = style({
	overflow: ['auto', 'overlay']
})
```

**Reply by MaxDonchenko** [reply]
Oh, that's nice! Thanks a lot!

**Reply by MaxDonchenko** [reply]
Works like a charm! Thanks for the quick response!

![image](https://user-images.githubusercontent.com/10671879/146563219-e2afbd9b-e99c-4318-bdf6-9859f7930dfc.png)
